### ISSUE TITLE
Virtual keyboard e vs e in italics

### ISSUE THREAD

**Issue by rs-mobitech** [arnog/mathlive#2099]
I notice the keycaps are confusing

- When in the 123 keyboard
   - e in italics > "ExponentialE"
   
- When in the 2nd keyboard with the infinity symbol
   - e (not in italics) > "ExponentialE"
   
- When in the 2nd keyboard with the infinity symbol and the caps lock
   - e (not in italics) > “[“Exp”,2]
   
I'm not sure if this is necessarily a problem but pointing this out as it may be considered confusing.

**Comment by arnog** [comment]
I'm not sure I understand the third case. You're talking about the keycap that's at the top-right corner of the keyboard?

**Comment by rs-mobitech** [comment]
Yes, that's correct.  My confusion was that in one case the e (without italics) creates ExponentialE, and in the other keycap the e (in italics) creates  ExponentialE.  I'm wondering if you meant to have it that e and _e_, both did the same thing.

**Comment by arnog** [comment]
So, basically, the issue is that the "e" in the 123 keyboard is in italics?

**Comment by rs-mobitech** [comment]
Yes, that’s correct.

**Comment by arnog** [comment]
Fixed with d981076b9feb84e0605fb6380d1a8797242cf832
